liblepton/scheme/lepton/legacy-config.scm

```diff
@@ -8,27 +8,47 @@
 
   #:use-module  ( ice-9 format )
   #:use-module  ( ice-9 match )
+  #:use-module  ( ice-9 optargs ) ; define*
```

Comment:
`optargs` is not required for `define*`, just for `define*-public`.

Reply:
yes, thank you.